pom.xml

```diff
@@ -507,6 +509,13 @@
                 <artifactId>netty</artifactId>
                 <version>${netty3.version}</version>
             </dependency>
+
+            <dependency>
+                <groupId>org.apache.iceberg</groupId>
+                <artifactId>${iceberg.name}</artifactId>
+                <version>${iceberg.version}</version>
+                <scope>test</scope>
```

Comment:
How about define `scope` at sub modules `<dependency>` block? 
IMO, define `version` in  `<dependencyManagement>` and define `scope` at sub modules `<dependency>` make things more clear.

Reply:
can we have a lint tool  for this？

Reply:
Unfortunately, I haven't seen those tools.